website/docs/packages/reference/query-module.md

```diff
@@ -70,56 +82,70 @@ When enabled, this property displays a confirmation modal each time before refre
 
 ## JS module
```

Comment:
What if we change this heading 2 to "JS module instance" so that it's more clear in the documentation the difference between JS module and JS module instance?